.github/workflows/build.yml

```diff
@@ -270,7 +270,7 @@ jobs:
           password: ${{ secrets.ARTIFACTORY_TOKEN }}
 
       - name: Run Tests
-        run: mvn -B jar:jar jar:test-jar surefire:test -DtrimStackTrace=false -Dliquibase.sdk.testSystem.test=hub,${{ matrix.testSystem }} -Dliquibase.sdk.testSystem.acceptLicenses=${{ matrix.testSystem }} -Dtest=*IntegrationTest,*ExecutorTest -DfailIfNoTests=false
```

Comment:
I think taking "hub" out of here will make the integration tests get skipped, since they are currently written to expect a hub system to be there. 

We'll have to either remove the hub usage (whatever that was) from the integration tests so they are not skipped or figure out a way to keep a hub available for testing enough in this codebase for these tests to still pass.